main.js

```diff
@@ -30,7 +42,33 @@ define(function (require, exports, module) {
         LiveDevServerManager.registerServer({ create: _createServer }, 9001);
 
         // Turn preview iFrame On
-        browser.browse();
+        browser.init();
     });
-    
+
+    // Next, we wait until the LiveDevelopment module is initialized
+    // so we can safely swap our transport and launcher modules for
+    // the defaults.
+    function _configureIframeForUse() {
+        // Set up our transport and plug it into live-dev
+        PostMessageTransport.setIframe(browser.getBrowserIframe());
+        LiveDevelopment.setTransport(PostMessageTransport);
```

Comment:
Same deal here, I think we should be working with an instance object for PostMessageTransport, where we pass in the browser iframe ref in the constructor.